downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -113,8 +113,31 @@ It is now possible to provide custom formatting configurations to the Ballerina
 
 #### OpenAPI tool
 
+- Integrated OpenAPI client generation to `bal build` command, in here user can provide the openapi tool config details in to the Ballerina.toml file and generate the client using `bal build` command.
+Example:
```

Comment:
```suggestion

E.g.,

```

Reply:
We have used `Example:` in other places (https://github.com/ballerina-platform/ballerina-dev-website/pull/8884/files).
Shall we finalize a pattern and make it consistent?

Reply:
For this release note i will add `For example`